rllib/core/models/tests/test_catalog.py

```diff
@@ -173,6 +185,76 @@ def test_get_encoder_config(self):
         # TODO(Artur): Add support for composite spaces and test here
         # Today, Catalog does not handle composite spaces, so we can't test them
 
+    def test_get_action_dist_cls_dict(self):
+        """Tests if we can create a bunch of action distributions.
+
+        Action distributions are created from the base catalog class. Things this
+        test checks:
+            - Whether we output the correct action distributions classes.
+            - Whether we can instantiate the action distributions, query their
+                required input dimensions and sample from them.
+
+        """
+        action_spaces_dist_types_and_expected_cls_dicts = [
```

Comment:
Can you make these a named tuple? for example the boolean in index 1 is not clear :)

Reply:
what about discrete + deterministic.

Reply:
What do you mean? They are there, no?

Reply:
we have Box + True, Box + False, and Disc + None. I think we should have Disc + True and Disc + False. No?

Reply:
@kouroshHakha Do you want also to return a deterministic distribution in these cases? Because so far, we've always just used "_max(q_values)_" for that sort of stuff.